### ISSUE TITLE
[FEATURE] The `indent` setting to apply for wrapped HTML attributes. 

### ISSUE THREAD

**Issue by mraksdev** [djlint/djLint#1877]
- [x] I've searched the [issues](https://github.com/djlint/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

# Feature Request
## Description

Currently, the `indent` configuration option in `pyproject.toml` (or `djlint.toml`) correctly sets the indentation level for nested HTML tags. However, it does not control the indentation level for **wrapped attributes** within a single HTML tag. Wrapped attributes are always indented with a fixed value of 4 spaces, regardless of the `indent` setting.

## Expected Behavior

The `indent` setting should apply consistently to all indentation, including wrapped HTML attributes.  Or add a setting for wrapped HTML attributes

If I set `indent = 2` in my configuration, I expect the following output:

**Desired output:**
```html
<link
  rel="apple-touch-icon"
  sizes="180x180"
  href="{% static "img/fav/apple-touch-icon.png" %}"
/>
```
**Actual Behavior**
```html
<link rel="apple-touch-icon"
      sizes="180x180"
      href="{% static " img/fav/apple-touch-icon.png" %}">
```

**Comment by stan-janssen** [comment]
Note that your expected output is exactly what is shown [on the example page](https://djlint.com/docs/formatter/#after):

<img width="662" height="437" alt="Image" src="https://github.com/user-attachments/assets/4c64aac2-6363-4a98-9c26-98d99a7c2750" />

What were the settings used to create that example?

**Comment by mraksdev** [comment]
Maybe I am missing some settings for indentation. I set indent to 2 because by default it was doing 4 spaces and it still does in some cases

<img width="1422" height="878" alt="Image" src="https://github.com/user-attachments/assets/b248fbe3-12ca-4956-ae9c-e7fbd2889c01" />

**Comment by mraksdev** [comment]
I didn't find the option to fix this in the docs, though.
